runtime/libpgmath/lib/CMakeLists.txt

```diff
@@ -149,9 +149,11 @@ elseif(${LIBPGMATH_SYSTEM_PROCESSOR} MATCHES "aarch64")
   if(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")
     set(DEFINITIONS_L1 HOST_WIN MAXCPUS=256 MAXCPUSL=8 MAXCPUSR=8 TARGET_WIN
     TARGET_WIN_ARM64 WIN_ARM64 WINNT WIN64 TARGET_ARM64 _PGI_NOBUILTINS)
+    set(FLAGS_L1 "/fp:fast")
```

Comment:
Use this only if `clang-cl` is used. (There's a cmake variable to see if compiler has gnu cli or not)